Pokedex/Model/Pokemon/PokemonExpanded.swift

```diff
@@ -11,6 +11,13 @@ struct PokemonExpanded: Codable {
     
     let sprites: PokemonSprite
     let name: String
+    let number: Int
+    
+    enum CodingKeys: String, CodingKey {
+        case sprites
+        case name
```

Comment:
`nit`: You can use this convention:
```suggestion
        case sprites, name
```